### ISSUE TITLE
No axis

### ISSUE THREAD

**Issue by lpatiny** [zakodium/react-plot#241]
Currently the minimal code is something like

```jsx
<Plot width={300} height={300}>
  <LineSeries
    data={[
      { x: 1, y: 1 },
      { x: 2, y: 2 },
      { x: 3, y: 3 },
      { x: 4, y: 2 },
      { x: 5, y: 1 },
    ]}
  />
  <Axis position="bottom" />
  <Axis position="left" />
</Plot>
```

I think it would be nice that Axis are optional and that you are allow to enter

```jsx
<Plot width={300} height={300}>
  <LineSeries
    data={[
      { x: 1, y: 1 },
      { x: 2, y: 2 },
      { x: 3, y: 3 },
      { x: 4, y: 2 },
      { x: 5, y: 1 },
    ]}
  />
</Plot>
```

But I'm not sure what should be the result in such a case. Should the system by default add a left and bottom axis like in the original example

![image](https://user-images.githubusercontent.com/1484241/149378403-00b065c2-be29-43de-a698-5a78dce1d8de.png)

Or should the SeriesViewPort take the full space like 

![image](https://user-images.githubusercontent.com/1484241/149378560-74457d77-3aea-487d-8a6a-29331f3c3974.png)

**Comment by maasencioh** [comment]
For me, the most consistent should be the first example. Hide the axis should be an explicit action from the user.

**Comment by stropitek** [comment]
> For me, the most consistent should be the first example. Hide the axis should be an explicit action from the user.

Not rendering the axis by not putting it in the jsx is explicit, isn't it?

**Comment by stropitek** [comment]
I think not putting an axis could have a use case, for example to have a small quick preview of data in a table. Not sure that this is the library to do it with. There are other solutions that might be performing better by rendering a canvas, especially if you imagine a lot of small graphs to render in one page.

But still think it's a valid use case

**Comment by maasencioh** [comment]
> Not rendering the axis by not putting it in the jsx is explicit, isn't it?

For me it is not, because like that you are waiting for the default behavior, the difference is that I would do something like this for the other case

```tsx
Plot width={300} height={300}>
  <LineSeries
    data={[
      { x: 1, y: 1 },
      { x: 2, y: 2 },
      { x: 3, y: 3 },
      { x: 4, y: 2 },
      { x: 5, y: 1 },
    ]}
  />
  <Axis position="bottom" hide />
  <Axis position="left" hide />
</Plot>
```

**Comment by targos** [comment]
I opened https://github.com/zakodium/react-plot/pull/248
Automatic axes are created with the default behavior.

**Comment by stropitek** [comment]
Makes sense to have to write less code while preserving axis since it's what we want most of the time.

If we need to add support for not rendering axes we can decide later how to do it but I don't think it should be by rendering the `Axis` component. Probably better to add an option at the plot level and throw in Axis if the Plot says there should be no axis
